README.md

```diff
@@ -722,9 +723,11 @@ const shell = electron.shell
 Now, we'll listen for link clicks and ask them politely to open in a new window instead of stepping over our little application.
 
 ```js
-$(document).on('click', 'a[href^="http"]', (event) => {
-  event.preventDefault()
-  shell.openExternal(event.target.href)
+document.body.addEventListener('click', (event) => {
+  if (event.target.href && event.target.href.startsWith('http')) {
```

Comment:
consider using `event.target.matches('a[href^="http"]')` instead as I believe it is closer to the jquery implementation you are replacing